hudi-utilities/src/main/java/org/apache/hudi/utilities/config/KafkaSourceConfig.java

```diff
@@ -66,14 +66,6 @@ public class KafkaSourceConfig extends HoodieConfig {
       .markAdvanced()
       .withDocumentation("Schema to deserialize the records.");
 
-
-  public static final ConfigProperty<Long> KAFKA_FETCH_PARTITION_TIME_OUT = ConfigProperty
-      .key(PREFIX + "fetch_partition.time.out")
```

Comment:
We don't delete the config directly.  We usually first mark such config with `deprecatedAfter` and still allow it in the next release.

Reply:
yeah, let's put the combatibility in high priority.